src/observed.decorator.spec.ts

```diff
@@ -6,21 +6,61 @@ const testValue1 = 10;
 const testValue2 = 25;
 const testValue3 = 50;
 
-class TestClass {
+interface Test {
+    property: any;
+    property$: Observable<any>;
+}
+
+class TestClass implements Test {
     // @ts-ignore
     @Observed() property = testValue1;
     readonly property$!: Observable<number>;
 }
 
-class TestClassNoDefaultValue {
+class TestSubject {
+    // @ts-ignore
```

Comment:
NIT: we should attempt to change the linting so that we can drop these `// @ts-ignore` lines.